Makefile

```diff
@@ -48,7 +48,7 @@ clean-test: ## remove test and coverage artifacts
 	rm -fr htmlcov/
 
 lint: ## check style with flake8
-	flake8 bigchaindb tests
+	flake8 --max-line-length=119 bigchaindb tests
```

Comment:
for future ref: http://flake8.pycqa.org/en/latest/user/configuration.html#project-configuration